### DISCUSSION TITLE
Thank you notes

### DISCUSSION THREAD

**Discussion by Aspire1Inspire2** [naver/billboard.js#3840] [Show and tell]
I am in the middle of making a graphic centered website around the US Census API. After trying out the chart.js bubble chart, I found your library handling the chart much better. chart.js has some bugs in their end that I cannot get around.
I ended up plotting some of my graphics using billboard.js, here are two examples:
![图片](https://github.com/naver/billboard.js/assets/77554817/7cdef2e2-3cec-456a-b22a-198ecb453e3c)
![图片](https://github.com/naver/billboard.js/assets/77554817/ff28e777-3943-47bc-8667-4d04426ed1c8)

The website is still WIP, not released yet. But I will probably pop up early next year here again with you once it get released. You will definitely see some familiar plots from the site.

**Comment by netil** [comment]
Hi @Aspire1Inspire2! Checked this post little bit late, but your comment made my day!
Also is grateful that this project helped. 👍 

Thanks!
